src/neo4j/graph-data.js

```diff
@@ -0,0 +1,30 @@
+// Format of node data are as follows:
+// { id: 'element.association.dbPrefix'+ ':' + 'element.association.id', factoidId: 'element.id',
+// name: 'element.name', type: 'element.type', dbId: 'element.association.id',
+// dbName: "element.association.dbName"}
+export const nodeData = [
+    // MAPK6 data
+    {
+        id: 'ncbigene:5597', factoidId: '598f8bef-f858-4dd0-b1c6-5168a8ae5349', name: 'MAPK6',
+        type: 'protein', dbId: '5597', dbName: 'NCBI Gene'
```

Comment:
`dbId` is redundant I suppose

Reply:
same with `dbName`? I can get rid of both of them

Reply:
`dbName` should be the normalised namespace, like `ncbi`, if we store it at all.  We could remove `dbId` and `dbName` completely if we're confident we're not going to do queries on `dbName`, e.g. give me all proteins that are grounded to `ncbi` rather than `uniprot`.

Reply:
![Screen Shot 2023-02-14 at 11 27 33 AM](https://user-images.githubusercontent.com/66929920/218797772-571e9a40-66a1-4954-884c-36f0454a392c.png)

I got "NCBI Gene"  from the JSON file way back when I was just getting the values straight from those files. So I think it's pretty consistent

I think I will keep those two fields for now, but I can easily remove them when we're certain they're not necessary

Reply:
In striving for consistency in syntax,  in Factoid I'm going by:

- [NCBI Gene](https://registry.identifiers.org/registry/ncbigene)
  - `dbPrefix`: `ncbigene`
  - `dbName`: `NCBI Gene`
  - Local id pattern  (regex): `^\d+$`
  - Compact identifier: `ncbigene:^\d+$`
- [ChEBI](https://registry.identifiers.org/registry/chebi)
  - `dbPrefix`: `CHEBI`
  - `dbName`: `ChEBI`
  - Local id pattern (regex): `^CHEBI:\d+$`
  - Compact identifier: `CHEBI:^\d+$`